benchmarks/common.py

```diff
@@ -1,18 +1,29 @@
 import os
 import timeit
 
+"""
+if environment variable $MULTICORE is set, benchmarks will be run for
+n_jobs vals = (1, $MULTICORE). Otherwise for n_jobs vals = (1, -1)
+MULTICORE can be a list of values as well.
+Example: export MULTICORE=2,3,4,5
+"""
 
-# if environment variable $MULTICORE is set, benchmarks will be run for
-# n_jobs vals = (1, $MULTICORE). Otherwise for n_jobs vals = (1, -1)
-N_JOBS_VALS = [1, int(os.getenv('MULTICORE', -1))]
+N_JOBS_VALS = [1, ]
 
+try:
+    multicore = os.getenv("MULTICORE")
+    results = map(int, multicore.split(','), -1)
```

Comment:
You seem to convert all ' to ". I don't know why (and personally prefer '). If that's agreed upon with others this here needs change, too.